extra_model/_cli.py

```diff
@@ -31,3 +32,9 @@ def entrypoint(input_path: Path, output_path: Path, debug: bool = False) -> None
     except ExtraModelError as e:
         logger.exception(e) if debug else logger.error(e)
         sys.exit(1)
+
+
+@click.command()
+def entrypoint_setup():
```

Comment:
Don't forget to update README with how to use this command

Reply:
will do! the following PR implements documentation generation and more through documentation can be generated after that is merged 👍